### PR TITLE
Some plant mutations

### PR DESCRIPTION
**Pull request by BombasterDS**
## About the PR
Added some new plant mutations for botanists:
 - Glasstle | Galaxythistle mutation. Produce onehit glass weapon that can be welded to glass.  (So it's glass source like steel-cap). Contains 15 razorium in it.
- Golden apples | Apple mutation. Golden apples with doctors delight, so it'll help you to regenerate (i though about adding 1u of ichor in it, but it looks very OP)
- Papercane | Sugarcane mutation. You can grow paper.

## Why / Balance
Just though that are plants i want to see in game

## Technical details
No code, just proto

## Media

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
No

**Changelog**

:cl:
- add: Added new plant mutations for apple, sugarcane and galaxythistle

### CODE REVIEW

**Comment by Partmedia** [comment]
Thanks for the contribution. Overall this seems reasonable. I'm going to have to get some feedback on the weapon stats as well as the amount of tricordazine, so please stand by.

**Comment by UbaserB** [comment]
since when was there a botany label

**Comment by Anzarot121** [comment]
At 10 potency, one apple will heal 1 brute damage, 0.5 poison, and 1 burn. Perhaps there is too little tricodrazine? Poppy with 10 potency has 3 bicaridine.

**Comment by BombasterDS** [comment]
> At 10 potency, one apple will heal 1 brute damage, 0.5 poison, and 1 burn. Perhaps there is too little tricodrazine? Poppy with 10 potency has 3 bicaridine.

Earlier i think about adding ichor to it (0.5-1). I think i'll set minimum trico to 3

**Comment by UbaserB** [comment]
deus can make omnizine so maybe balance it around that?

**Comment by BombasterDS** [comment]
> deus can make omnizine so maybe balance it around that?

Apple can grow repeatly without trait, so you will have more apples + deus have desoxyephedrine so it'll make you more poison damage if you eat it to heal yourself

Golden apple is designed as light healing way for crew if the have low damage and want heal themself without going to med everytime

I really want to add ichor to it because of it's rarity, but even 1u of ichor will heal 10 brute, 10 burn and 5 poison what looks very OP

**Comment by thebadman4662** [comment]
If omnizine is not being used because normal meds are already strong enough to keep it only as Necrozol ingredient then I do not see how 1u ichor is in any way OP. Do not forget all that healing will get split by subdamages.

Space Dragon gets 8u of ichor for every crew mob they eat, thats barely more than sinble bullet wound while 5u injection brute meds will heal almost twice as much and can be mass produced.

**Comment by MrRiddlz** [comment]
If I can give my two cents.

Doctors apples - Add basic meds (bic, kelo, Dylo ect) As the old saying goes "An apple a day keeps the Doc away"
Make sure the apples them selves have a higher then normal scaling on the regents per potency. Keeps People like me from abusing the shit out of it.

If you want to have ichor in a plant consider reskinning the Dues sprite to Gaia (Ref ss13 paradise plant). This ensures that you have to go through another cycle of plant change ie:

Vulga -> Dues -> Gaia

This will keep it out of the hands of botanists till mid game since they have to brave the gauntlet of RNG a second to for their plant. Prevents round start access.

**Comment by BombasterDS** [comment]
> ...I always dreamed of a Pink Apple with docs delight ...because yea ..."An apple a day keeps the Doc away" cant make better example unlike tricords , docs delight will heal a bit more but with a drawback it makes you slightly drunk and also more hungry. Maybe someone might makes a different mutation for this . But to the golden apple think its okay with just the tricords , still makes it a great alternative for some quick food and healing...maybe you could consider adding docs delight to it instead of ichor if the tricords alone feel not enought (or maybe as juicing option so barkeep might snatch also a few)

I forgot about doctor's delight! I believe replacement trico with it is actually good idea.

**Comment by Aisu9** [comment]
> > ...I always dreamed of a Pink Apple with docs delight ...because yea ..."An apple a day keeps the Doc away" cant make better example unlike tricords , docs delight will heal a bit more but with a drawback it makes you slightly drunk and also more hungry. Maybe someone might makes a different mutation for this . But to the golden apple think its okay with just the tricords , still makes it a great alternative for some quick food and healing...maybe you could consider adding docs delight to it instead of ichor if the tricords alone feel not enought (or maybe as juicing option so barkeep might snatch also a few)
> 
> I forgot about doctor's delight! I believe replacement trico with it is actually good idea.

Doctor's Delight is quite strong to be present in high quantity no?

**Comment by BombasterDS** [comment]
> > > ...I always dreamed of a Pink Apple with docs delight ...because yea ..."An apple a day keeps the Doc away" cant make better example unlike tricords , docs delight will heal a bit more but with a drawback it makes you slightly drunk and also more hungry. Maybe someone might makes a different mutation for this . But to the golden apple think its okay with just the tricords , still makes it a great alternative for some quick food and healing...maybe you could consider adding docs delight to it instead of ichor if the tricords alone feel not enought (or maybe as juicing option so barkeep might snatch also a few)
> > 
> > 
> > I forgot about doctor's delight! I believe replacement trico with it is actually good idea.
> 
> Doctor's Delight is quite strong to be present in high quantity no?

I decreased the numbers to 3-13 with 10 potencyDivisor. I think it sounds balanced

**Comment by BombasterDS** [comment]
> dont forget it gets split the healing so 1u docs delight 2 brute 2 burn 2 poison will actually only heal like 0.65 slash or so around so 10u docs delight (20 brute) would only be around 6.5 slash healing + 1u docs delight still comes with -1.2 satiation (from guidebook) makes you more hungry and the drunk downside if you have like 50u at once or so (1u docs will metabolize in 0.1 ethanol) so if you wanna just fully heal with docs...you gonna be very drunk and hungry (which actually is a small bonus for dwarfes) if you consume it as pure drink with no food , the only real problem with docs delight is basicly at some point you run out of cream to make it since i am not sure if there is a way to make cream ( the juices all of botany can make and tricords from medical, but cream...yea no only restocks sadly)

Small hegen regen is actually balanced thing:
1. Apples is repeatable, so you'll always will have more apples then ambosia or poppy.
2. Delight heals almost all damage in the game without trico's limit in <50 damage, so if you'll have large amount of this apples you'll be able to heal yourself from (almost) any situation
3. Golden apples don't have negative reagents like ambrosia, which applies poison damage when you eat it raw. (Nutriments and vitamins will almost overcome delight's negative saturation, so i don't count it as negative thing). Apples designed more like light crew treat tool than chemist's way to get more meds. 

I believe that this amount of delight is good for golden apples.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 1]
```diff
@@ -873,6 +888,42 @@
...
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 30
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 10
+        - ReagentId: Vitamin
+          Quantity: 4
+        - ReagentId: DoctorsDelight
```
Comment (IProduceWidgets) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:910]: Just make this trico, DD is a mixed drink and this doesn't make any sense. Its also got no limit like trico does.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 2]
```diff
@@ -95,6 +95,21 @@
...
+  parent: ProduceBase
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Hydroponics/papercane.rsi
+  - type: SolutionContainerManager
+  - type: Produce
+    seedId: papercane
+  - type: Log
+    spawnedPrototype: SheetPaper1
+    spawnCount: 2
```
Comment (IProduceWidgets) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:111]: I have no issue with a plant that makes paper, but I do wonder if it wouldn't make more sense to just have a way to turn logs / wood into paper instead though?

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 3]
```diff
@@ -1415,6 +1466,71 @@
...
+        acts: [ "Destruction" ]
+  - type: DamageOnHit 
+    damage:
+      types:
+        Blunt: 10
+  - type: MeleeWeapon
+    wideAnimationRotation: 60
+    damage:
+      types:
+        Slash: 15
```
Comment (IProduceWidgets) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:1526]: For context, this is already stronger than all knives, and this is the same slash damage the captain's sabre does. Thats before the razorium, which if I understand correctly between 1-25u with spawned ones at 15u. razorium is ~1/6th of Unstable Mutagen (which crits with 20u) and also does bleed.

tl;dr I think its a bit strong.

**Comment by IProduceWidgets** [comment]
Disclaimer that I cannon merge your PR.

**Comment by UbaserB** [comment]
Still doesnt make sense imo

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml (hunk 3 above)
Comment (BombasterDS) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:1526]: Razorium don't inject in body after hit + this weapon will break after one hit. Death nettle will be more dangerous than this.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml (hunk 2 above)
Comment (BombasterDS) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:111]: Well, i really don't like multiple resources source (when you can get wood and paper for one plant). So in my opinion - tower cap for wood only and paper cane for paper only.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml (hunk 1 above)
Comment (BombasterDS) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:910]: Poppy have bicaridine that mixed chem. I don't think that it makes so much trouble. DD just fit more.
